content/education/manage-coursework-with-github-classroom/teach-with-github-classroom/leave-feedback-with-pull-requests.md

```diff
@@ -24,6 +24,9 @@ To create and access the feedback pull request, you must enable the feedback pul
 {% data reusables.classroom.sign-into-github-classroom %}
 1. In the list of classrooms, click the classroom with the assignment you want to review.
 {% data reusables.classroom.click-assignment-in-list %}
+1. For the student to review press the feedback button to the right of the student row.
+1. On the Feedback PR select the Files Changed tab.
+2. Annotate, review and submit commentary for the student.
 1. To the right of the submission, click **Review**.
 1. Review the pull request. For more information, see "[AUTOTITLE](/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request)."
```

Comment:
```suggestion
1. Each student will have a submission row, press the Feedback link to navigate to the student's pull request.
1. On the Feedback PR select the Files changed tab.
1. Review the pull request. For more information, see "[AUTOTITLE](/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request)."
```

Sharing a recording of what this flow currently looks like: 

https://github.com/github/docs/assets/1108003/8674f822-475e-4ac7-9ab1-0447aae4e2fa